README

```diff
@@ -94,6 +95,58 @@ running the following script:
 
     echo please help | gcc -fplugin=libfa.so -fplugin-arg-libfa-help -xc -
 
+Building from sources against the system LLVM
```

Comment:
Please make it clear that the LLVM support is experimental and wrap the lines at 80th column.